turbinia/workers/artifact.py

```diff
@@ -64,6 +64,10 @@ def run(self, evidence, result):
         '--artifact_filters',
         self.artifact_name,
     ]
+
+    if not config.DOCKER_ENABLED:
+      cmd.insert(0, 'sudo')
```

Comment:
Is sudo really required to run image_export.py? Why doesn't the docker container need to run it with root privileges? Since we are running the workers with ```--privileged``` the worker program should be able to access block devices no?

Reply:
I'm not exactly sure why sudo wouldn't be needed in the docker container, but maybe the commands run as root by default?  I'm not 100% sure about the details of how the `--privileged` flag works, but my understanding is that it gives the container the ability to run privileged commands, but AFAIK not every user/process in the container is automatically root equivalent without running as the root user or with sudo, but I could be wrong.

Reply:
We run into 2 things here:
* not all container have sudo installed (eg the plaso container does not)
* not all containers run their binary as user `root`, eg we run the worker as the `turbinia` user

In the specific case of image_export.py:
* in the turbinia-worker we need sudo as the worker runs as the `turbinia` user
* in the plaso container the binary runs as root and there is no `sudo`

The privileged flag is not related (privileged has more to to with access to devices etc, which image_export needs but is not related to `sudo` usage)